admin_frontend/src/components/ApiKeys.jsx

```diff
@@ -211,6 +227,25 @@ const ApiKeysPage = () => {
 									fullWidth
 								/>
 							</TableCell>
+							<TableCell>
+								<Checkbox
+									checked={txnMode === 0? false : true}
+									onChange={handleChange}
+									inputProps={{ 'aria-label': 'controlled' }}
+								/>
+							</TableCell>
+							<TableCell>
+							<TextField
+									type="number"
+									variant="outlined"
+									color="secondary"
+									label="NO_OF_TRANSACTIONS_IN_A_MONTH"
+									onChange={(e) => setNoOfTxn(e.target.value)}
+									value={noOfTxn}
+									required
+									fullWidth
+								/>
```

Comment:
Indentation is way off - do you have a linter / code formatter?

Reply:
yes will do that